views/partials/jira-configuration-error-summary.hbs

```diff
@@ -10,25 +10,27 @@
         You could continue to use the integration and still view the backfilled data in Jira, while the app attempts to reprocess these entities.</p>
     </div>
 
-    <div class="error-summary-border-bottom">
-      <div class="error-summary-errors-header error-summary-border-bottom">
-        Description
-      </div>
+    <div class="error-summary-border-top">
+      <h4 class="error-summary-errors-header">
+        Please resolve the following errors:
+      </h4>
+      <ul>
       {{#if syncErrors.PERMISSIONS_ERROR}}
-      <div>
+      <li>
         <b>{{syncErrors.PERMISSIONS_ERROR}}</b> tasks have not been backfilled. Please
         <a href="{{html_url}}"
            data-installation-link="{{html_url}}"
-           target="_blank">accept permissions</a>
-        on GitHub (you must be the admin of your organization), then retry backfill.
-      </div>
+           target="_blank">accept permissions on GitHub</a>
+         (you must be the admin of your organization), then retry backfill.
+      </li>
       {{/if}}
 
       {{#if syncErrors.CONNECTION_ERROR}}
-      <p>
-        We were unable to complete backfill for <b>{{syncErrors.CONNECTION_ERROR}}</b> repositories, because something unexpected occurred, please retry the backfill.
-      </p>
+      <li>
+        We were unable to complete backfill for <b>{{syncErrors.CONNECTION_ERROR}}</b> repositories, because something unexpected occurred, please retry the backfill or rasie a support ticket.
```

Comment:
A couple of things here:

1. It doesn't look like there's any number in the screenshot you attached so it just says: 'We were unable to complete backfill for repositories'.
2. Edit: We were unable to complete backfill for <b>{{syncErrors.CONNECTION_ERROR}}</b> repositories because something unexpected occurred. Please retry the backfill or raise a support ticket. (removed comma before because, started a new sentence at 'please' and corrected rasie typo)

Reply:
ahh yea, that wasnt a fair screenshot since i just forced it to show, so should be a number, ill re-screenshot
but yup agree will conditional on the 1 vs 1+
I accept the copy change too!